addons/point_of_sale/static/src/js/models.js

```diff
@@ -3142,7 +3144,7 @@ export class Order extends PosModel {
             if (lineChange["quantity"] > 0 && !cancelled) {
                 toAdd.push(lineChange);
             } else {
-                lineChange["quantity"] *= -1; // we change the sign because that's how it is
+                lineChange["quantity"] = Math.abs(lineChange["quantity"]); // we need always positive values.
```

Comment:
Can you quickly explain me why it's sometimes negative? I feel like we are just masking the problem by taking the absolute value.